endpoints/openrtb2/auction.go

```diff
@@ -107,7 +126,10 @@ func (deps *endpointDeps) Auction(w http.ResponseWriter, r *http.Request, _ http
 	// That status code can't be un-sent... so the best we can do is log the error.
 	if err := enc.Encode(response); err != nil {
 		glog.Errorf("/openrtb2/auction Error encoding response: %v", err)
+		ao.Errors = append(ao.Errors, fmt.Errorf("/openrtb2/auction Error encoding response: %v", err))
 	}
+
+	deps.analytics.LogAuctionObject(&ao)
```

Comment:
This would be called twice due to [the deferred call above](https://github.com/prebid/prebid-server/pull/418/files#diff-09e5d415f33d74fd6df20b2584f318d1R67).

Reply:
you're right! updated the code.